src/toil/cwl/cwltoil.py

```diff
@@ -82,6 +82,8 @@
 from toil.common import Config, Toil, addOptions
 from toil.version import baseVersion
 
+logger = logging.getLogger(__name__)
```

Comment:
Why not use `_logger` from https://github.com/DataBiosphere/toil/blob/dd7322725bac8fb65aacd70c3f3514982a2c8b2d/src/toil/cwl/cwltoil.py#L68 ?

Reply:
I set this up the way other Toil components do logging. I've never done logging the cwltool way before.

Presumably we should use only one of these in this module.

I actually would favor the Toil style as the right one to use here. The [cwltool logger setup](https://github.com/common-workflow-language/cwltool/blob/master/cwltool/loghandler.py) is pretty simple; it hardcodes INFO as the log level, logs everything as coming from "`cwltool`", and sends it to a default stream handler. The Toil style is to have loggers that report what file is doing the logging, and it obeys some options (`--logDebug`, `--logInfo`, etc.) to let the user control the log level.

Since Toil's `cwltoil.py` isn't actually `cwltool`, I think it makes sense for it to not log via `cwltool`'s internal logger. Moreover, since I'm adding some debug-level messages, I need a logger that I can set to debug level from the command line to actually see them, and the Toil logger setup has that, while I don't think cwltool's does.

Should Toil maybe be doing something to cwltool's `_logger` to adopt it and get its log messages to go through the same stream handler as Toil's?

Reply:
Yeah, if I send my debug-level logging to `cwllogger`, I don't see it even when running with `--logDebug`.